keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/ArtifactRepositoryTests.kt

```diff
@@ -610,7 +610,6 @@ abstract class ArtifactRepositoryTests<T : ArtifactRepository> : JUnit5Minutests
             )
             expect {
               that(envArtifactSummary).isNotNull()
-              that(envArtifactSummary?.isPinned).isTrue()
```

Comment:
In order to keep the same functionality, should we add here (and below, if I understand the change correctly):
` that(envArtifactSummary?.pinned).isNotNull()` ?

Reply:
The next line does an `isEqualTo` on the same field so testing it's not null explicitly doesn't really add anything.